File-Storage-Security/Deployment/aws-nested-aio-promote-quarantine-plugin/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Deploy the All-In-One stack with Plugin Promote and Quarantine.
+This script will deploy the File Storage Security AIO Stack with the promote/quarantine lambda functionality as a nested stack. 
+
+Launch the CloudFormation template provided below.
+[![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=c1-fss-aio-v2&templateURL=https://aws-workshop-c1as-cft-templates.s3.amazonaws.com/aws-aio-nested-w-plugin.yaml)
+
+
+## Prerequisites
+
+1. **Create S3 buckets**
+    - Create a 'Scanning bucket' to be monitored by File Storage Security.
+    - Create a 'Promote bucket' to receive clean files. Example: `fss-promote`.
+    - Create a 'Quarantine bucket' to receive quarantined files. Example: `fss-quarantine`.
+
+## Stack Parameters
+
+Fill out the Quick create stack page as follows:
+
+- Stack name: Specify the name of the stack. Example: FileStorageSecurity-All-In-One.
+- S3BucketToScan: Specify the name of your S3 bucket to scan, as it appears in S3. You can only specify one bucket.
+- ScannerEphemeralStorage: The size of the scanner lambda function's temp directory in MB. The default value is 512, but it can be any whole number between 512 and 2048 MB. Configure a large ephemeral storage to scan larger files in zip files.
```

Comment:
Is that really necessary to mention that here?

I was thinking if is more logical to mention what conditions the template is expecting your bucket to be/have, thoughts?

Reply:
I just used what was in the documentation